components_new/query_handler/queryHandler.py

```
@@ -0,0 +1,46 @@
+    return response
+
+async def receiveMessage(websocket):
+    '''
+        Handles messages from websocket and echos a response.
+    '''
+    async for message in websocket:
+        try:
+            message = json.loads(message)   
+            response = handleQuery(message=message)   
+            await websocket.send(json.dumps(response))  
+        except json.JSONDecodeError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)